peachjam/views/legislation.py

```diff
@@ -130,15 +130,23 @@ def get_timeline_events(self):
 
         publication_date = self.object.metadata_json.get("publication_date", None)
         if publication_date:
+            api_url = "https://api.laws.africa/v2/"
+            commons_url = "https://commons.laws.africa/"
+            publication_url = (
```

Comment:
You don't need the if and else. The .get will return None. 

Clever approach to fixing the url :)